sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -237,6 +253,10 @@ def start_k8s_software(self):
         """)
         self.host_node.remoter.run(f"sudo -E bash -cxe '{script}'")
 
+    def on_deploy_completed(self):
+        if self.scylla_image:
+            self.docker_pull(self.scylla_image)
```

Comment:
I think that we should do the same on minikube too. The same - upload docker image onto nodes.
We need following command here:
`$ minikube cache add %image%`
And viewing it:
`$ minikube cache list`

Reply:
we are running minikube in driver=none mode, in this mode docker pull is enough.

Reply:
added